content/en/development/design-docs/blb-migration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Basic to Standard Load Balancer Migration"
+linkTitle: "Basic to Standard Load Balancer Migration"
+type: docs
+description: >
+    Support migration from Basic Load Balancer to Standard Load Balancer.
+---
+
+## Background
+
+On September 30, 2025, Basic Load Balancer will be [retired](https://learn.microsoft.com/en-us/azure/load-balancer/skus). This design document describes the planned migration path from Basic Load Balancer to Standard Load Balancer.
+
+## Design
+
+> The following `User` stands for human users, or any cluster provisioning tools (e.g., AKS).
+
+1. User must manually decouples all virtual machines from the Basic Load Balancer backend pool before migration.
```

Comment:
could we do this in our codes? it should be possible as we already have the cleanup logic (need some changes to preserve IP)

Reply:
Sure, as we can have the service object so we know the ip information, no need to have another way to preserve IP.